### PR TITLE
Footermod

### PR DESCRIPTION
**Pull request by snschau**
# Summary of changes
Modify Custom Footer to add the social media icons for ovc footers 
check to show that non ovc footers are not effected

## Frontend
- Add React Icons to packages
- Add footer-social.js (to blocks/ovc)
  - custom code to add social media icons
- Modified customFooter.js to check if unit tag is OVC and if so - add social icons below the image.
- Add social-icons.css - to remove some styles that were interfering with how icons looked  

 
## Backend
- None

# Test Plan

https://ovccontenthub.gatsbyjs.io/ovc/gceb3 icons at the bottom should work as on https://ovc.uoguelph.ca/
https://ovccontenthub.gatsbyjs.io/test-media-text-in-section - a test page to show that if not an ovc footer - icons don't display
https://ovccontenthub.gatsbyjs.io/cancer-to-delete (or any page that does not have a footer) no change or footer elements added

### CODE REVIEW

File: src/styles/ovc/social-icons.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.text-dark-social {
+    color: #343a40!important;
+}
+
+.text-dark-social:hover,
+.text-dark-social:focus {
+    color: #121416!important;
+    background: none !important;
+    text-decoration-color: none !important;
```
Comment (tqureshi-uog) [inline src/styles/ovc/social-icons.css:9]:
```suggestion
    text-decoration: none !important;
```

I'm noticing an underline flash whenever you hover over an icon and then move away. It also appears when you tab from one icon to the next. This is on Windows in both Chrome and Firefox. Perhaps the above suggestion might resolve this.

Comment (snschau) [reply src/styles/ovc/social-icons.css:9]: thanks!

Comment (snschau) [reply src/styles/ovc/social-icons.css]: have tried that change and it still is there.

**Comment by snschau** [comment]
Abandoned in favour of the one modified by Miranda.

**Pull request closed without merging**